### ISSUE TITLE
Support for Typesense sync target

### ISSUE THREAD

**Issue by Jaakkonen** [long2ice/meilisync#15]
The code-base seems to have quite good abstractions for making it possible to swap out Meilisearch from the target sync system for other search-tailored document databases. I came across this project while looking for Postgresql -> Typesense or Postgresql -> Meilisearch syncing tools but would prefer to use [typesense](https://typesense.org/) as my search database due to its [vector search](https://typesense.org/docs/0.24.1/api/vector-search.html) capability. Currently this syncing is achievable by using [Airbyte](https://github.com/airbytehq/airbyte) but that's way too big overkill and hard to adopt compared to this project.

**Comment by long2ice** [comment]
Yes, I plan to split a independent project just produce change data stream, so that we can choice where the data to destination. But need some time.

**Comment by sanders41** [comment]
@Jaakkonen just an FYI, Meilisearch is adding vector search in v1.3.0 (currently in RC). I haven't used it in Meilisearch or Typesense so I can't say if one is better than the other, but thought you might be interested.
